src/jukebox/components/rfid/hardware/rc522_spi/rc522_spi.py

```diff
@@ -87,7 +86,7 @@ def __init__(self, reader_cfg_key):
                                    pin_rst=pin_rst,
                                    pin_irq=pin_irq,
                                    antenna_gain=antenna_gain,
-                                   pin_mode=GPIO.BCM)
+                                   pin_mode='BCM')
```

Comment:
Does this accept strings?

Reply:
Yep, [see](https://github.com/hoffie/pi-rc522-gpiozero/blob/1dc878c8a3d6f32ad6601877b00e7316d72811d1/pirc522/rfid.py#L13).
 It would work also with `GPIO.BCM` but i wanted to eliminate the import and give the possibility to use other gpio libs for gpiozero.